### PR TITLE
Feature/auth

### PR DESCRIPTION
**Pull request by TriangleJuice**

### CODE REVIEW

File: config/options.js [hunk 1]
```diff
@@ -8,22 +8,22 @@ const generatorNames = Object.keys(generators);
...
     param: '-S, --no-setup',
     description: 'Skip setup questions',
   },
-  {
-    param: '-f, --frontend <frontend>',
-    description: 'Frontend framework (React or Angular)',
-    validation: /^(react|angular)$/i,
-    fallback: 'react',
-  },
+  // {
```
Comment (vademo) [inline config/options.js]: Remove useless commented code. If we implement this later we can easily add this config.

File: generators/nodejs.js [hunk 2]
```diff
@@ -77,15 +79,14 @@ async function copyBaseProject() {
...
-  log(chalk`Cleanup tmp folder.`);
+  debug.logger(`Cleanup tmp folder`);
   deleteFolderRecursive('./tmp');
-  log(chalk`{green Done. }`);
 }
 
 async function setDB(db) {
   if (db === 'mongodb') {
-    log(chalk`{yellow.bold Installing MongoDB }`);
+    log(chalk.green.bold(`
```
Comment (vademo) [inline generators/nodejs.js]: Is there an advantage to use this syntax ?

File: generators/react.js [hunk 3]
```diff
@@ -45,14 +46,14 @@ const questions = [
     name: 'routing',
     message: 'Do you want to add basic routing?',
     default: true,
-  },
+  }
```
Comment (vademo) [inline generators/react.js]: our linting prefers trailing ","

File: index.js [hunk 4]
```diff
@@ -44,27 +54,32 @@ async function askQuestions() {
...
 async function run() {
-  fancyLog('blue.bold', `Welcome to the Digipolis starter kit! (v${pjson.version})`, '=');
+  fancyLog('cyan.bold', `Welcome to the Digipolis starter kit! (v${pjson.version})`, '=');
   if (program.setup) {
     const config = await askQuestions();
     if (program.debug) {
       debug.enable();
     }
     const { frontend, backend } = config;
+    // console.log(config);
```
Comment (vademo) [inline index.js]: leftover from debug?

File: generators/nodejs.js (hunk 2 above)
Comment (TriangleJuice) [reply generators/nodejs.js]: Yep, it inserts an empty line, which makes it more readable while installing.

File: config/options.js (hunk 1 above)
Comment (TriangleJuice) [reply config/options.js]: Fixed.

File: generators/react.js (hunk 3 above)
Comment (TriangleJuice) [reply generators/react.js]: Fixed.

File: index.js (hunk 4 above)
Comment (TriangleJuice) [reply index.js]: Fixed.

File: generators/nodejs.js (hunk 2 above)
Comment (vademo) [reply generators/nodejs.js]: but its ugly in code 🤢

Comment (TriangleJuice) [reply generators/nodejs.js]: True… I will replace it with a helper function, so you only have to look at it once 😇

Comment (vademo) [reply generators/nodejs.js]: 😁

File: generators/nodejs.js [hunk 5]
```diff
@@ -64,9 +49,10 @@ function getOptions() {
 async function copyBaseProject() {
   const { baseProject } = nodeConfig;
   const { repository, branch } = baseProject;
-  log(chalk`{blue 🔗 Clone backend verion: {yellow.bold ${branch}}}`);
+  debug.logger(`Clone backend version: ${branch}`);
+  // log(chalk`{blue 🔗 Clone backend version: {yellow.bold ${branch}}}`);
```
Comment (lievenvg) [inline generators/nodejs.js]: Comment still needed?

File: generators/nodejs.js [hunk 6]
```diff
@@ -64,9 +50,10 @@ function getOptions() {
...
```
Comment (TomVanDenBroeck) [inline generators/nodejs.js]: still needed?

**Pull request merged**